src/GitHub.VisualStudio/UI/Views/Controls/PullRequestListItem.xaml

```diff
@@ -110,6 +110,7 @@
                 Command="{Binding OpenPROnGitHub, RelativeSource={RelativeSource FindAncestor, AncestorType={x:Type local:PullRequestListView}}}"
                 CommandParameter="{Binding Number}"
                 Content="{Binding Number}"
+                ToolTip="View Pull Request on GitHub"
```

Comment:
Well, that would make the PR a little more interesting. 😉 

I was thinking GitHub the application as opposed to github.com the site. Can I claim president for this interpretation?

![image](https://cloud.githubusercontent.com/assets/11719160/22521964/b53e03ae-e8b1-11e6-840b-c14661c54dc2.png)
(the ghe.io search also says "Search GitHub")

My instinct would be to KISS. What do you think?